walnuts_cpp/include/walnuts/nuts.hpp

```diff
@@ -153,23 +153,24 @@ Span<S> build_leaf(const F &logp_grad_fun, const Span<S> &span,
   Vec<S> rho_next;
   Vec<S> grad_theta_next;
   S logp_theta_next;
-  if constexpr (D == Direction::Forward) {
+  if constexpr (D == Direction::Forward)
     leapfrog(logp_grad_fun, inv_mass, step, span.theta_fw_, span.rho_fw_,
              span.grad_theta_fw_, theta_next, rho_next, grad_theta_next,
              logp_theta_next);
-  } else {  // Direction::Backward
+  else // Direction::Backward
```

Comment:
Aside: I'm generally also on team always-braces, but in this particular case I think we may also want to look into having this logic be internal to leapfrog (accepting D as a tparam) similar to walnut's macro step code

Reply:
I can live with always braces. I think it's a broader consensus.  I generally prefer any decision that leads to less vertical space "wasted."

Reply:
There's a good case to be made to keep the code as parallel as possible even though the leapfrog algorithm is usually coded much more simply than we already have.